python/dgl/nn/pytorch/gt/spatial_encoder.py

```diff
@@ -1,12 +1,15 @@
 """Spatial Encoder"""
 
-import math
-
 import torch as th
 import torch.nn as nn
 import torch.nn.functional as F
 
-from ....batch import unbatch
+
+def gaussian(x, mean, std):
+    """compute gaussian basis kernel function"""
+    const_pi = 3.14159
```

Comment:
Is this the choice used by the original implementation?

Reply:
yes